client/src/App.tsx

```diff
@@ -40,12 +41,13 @@ function App(): JSX.Element {
           <Switch>
             <Route exact path="/login" component={Login} />
             <Route exact path="/signup" component={Signup} />
+            {/* <Route exact path="/calendar" component={MyCalendar} /> */}
             <Route exact path="/dashboard">
               <Dashboard />
             </Route>
-            <Route path="*">
+            {/* <Route path="*">
               <Redirect to="/dashboard" />
-            </Route>
+            </Route> */}
```

Comment:
Same here, we should try making sure all the routes work with the feature so we aren't commenting important parts of the app out.

Reply:
fixed.